lib/Bitreserve/Command/CreateTokenCommand.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+use Bitreserve\BitreserveClient;
+use Bitreserve\Exception\AuthenticationRequiredException;
+use Bitreserve\Exception\BadRequestException;
+use Bitreserve\Exception\TwoFactorAuthenticationRequiredException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class CreateTokenCommand extends Command
+{
+    /**
+     * Configure the command.
+     */
+    protected function configure()
+    {
+        $this->setName('token:create')
```

Comment:
I suggest renaming this to `tokens:create`. Keep in mind that we will add `tokens:list` and `tokens:revoke`.